benchmark/C/Savina/src/micro/Counting.lf

```diff
@@ -38,7 +38,13 @@
  ]]] */
  // [[[end]]]
 
-target C;
+target C{
+    /* [[[cog
```

Comment:
Could you update the mechanism for setting the number of threads with the one I introduced in #485? This applies of course to all the benchmarks, not just this one.